File: Python/internationaldates.py
# Quick solution to: https://open.kattis.com/problems/internationaldates

date = list(map(int,input().split("/")))
if(int(date[0]) > 12):
    print("EU")
elif(int(date[1]) > 12):
    print("US")
else:
    print("either")

File: Javascript (NODE JS)/internationalDates.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});


rl.on('line', (line) => {
    split = line.split("/")
    date = []
    for(i = 0; i < split.length; i++){
        date.push(parseInt(split[i]))
    }
    if(date[0] > 12){
        console.log("EU")
    }
    else if(date[1] > 12){
        console.log("US")
    }
    else{
        console.log("either")
    }
});
